datafusion/expr/src/expr.rs

```diff
@@ -186,7 +186,16 @@ pub enum Expr {
 
 #[derive(Clone, PartialEq, Eq, Hash, Debug)]
 pub struct Unnest {
-    pub exprs: Vec<Expr>,
+    pub expr: Box<Expr>,
```

Comment:
In the expression position, only one argument can be accepted.

Reply:
https://www.postgresql.org/docs/current/functions-array.html
It was Vec since there are multi-columns in postgres.

`select * from unnest(ARRAY[1,2], ARRAY['foo','bar','baz']) as x(a,b) →`